client/my-sites/importer/newsletter/subscribers.tsx

```diff
@@ -62,14 +62,11 @@ export default function Subscribers( {
 					click 'Export.' Once the CSV file is downloaded, upload it in the next step.
 				</p>
 				<Button
-					variant="secondary"
 					href={ `https://${ fromSite }/publish/subscribers` }
 					target="_blank"
 					rel="noreferrer noopener"
-					iconPosition="right"
-					icon={ external }
 				>
-					Export subscribers
+					Export subscribers <Gridicon icon="external" />
```

Comment:
I decided to use the Gridicons approach since the combination of `icon={ external }` and `iconPosition="right"` looks a bit off:			

<img width="748" alt="Screenshot 2024-09-02 at 14 37 28" src="https://github.com/user-attachments/assets/fe06c346-06ac-44f7-9c0c-2d5f070f4272">

Noticed in another place we are rendering the icon on the left (which looks ok), so maybe let's do this for the importer as well.

<img width="748" alt="Screenshot 2024-09-02 at 14 44 40" src="https://github.com/user-attachments/assets/462c6b5c-fa85-403c-b3fe-109fee0df0dd">
<br><br>

Should we mark `Gridicon` as deprecated? I was not aware we don't like it anymore.

Reply:
It's not deprecated per-se, just there's preference to lean to core componentry where possible. Only Dashicons set is truly deprecated. (PuPv3-aHW-p2)